### DISCUSSION TITLE
基于 NebulaGraph 图数据库的 ID Resolution 方法与代码示例 - siwei.io

### DISCUSSION THREAD

**Discussion by giscus** [siwei-io/siwei-io.github.io#12] [Comments]
# 基于 NebulaGraph 图数据库的 ID Resolution 方法与代码示例 - siwei.io

本文是一个基于 NebulaGraph 上的图算法、图数据库、图神经网络的 ID-Mapping 方法综述，除了基本方法思想的介绍之外，我还给大家弄了可以跑的 Playground。

https://www.siwei.io/identity-resolution/

**Comment by zsj0515** [comment]
初学者如何把demo跑起来呢

**Reply by wey-gu** [reply]
我最近把它放到 studio 的 starter dataset 里了，在 studio 的 welcome 页点下载就可以了

![image](https://github.com/siwei-io/siwei-io.github.io/assets/1651790/5ec7a58c-3d39-433d-8345-2205465d4f09)

**Reply by zsj0515** [reply]
老师这是版本几，在3.2.2的studio请求/welcome没有

**Reply by wey-gu** [reply]
3.6.2 可以哈，你可以装一个，3.6.2 和 3.2.2 可以并存的，你如果用 docker desktop 里的 NebulaGraph 插件，安装，默认自带两个版本 studio。
